cfgov/jinja2/v1/owning-a-home/loan-estimate/index.html

```diff
@@ -0,0 +1,106 @@
+{% extends "base.html" %}
+{% import "molecules/social-media.html" as social_media with context %}
+{% set page={"search_description":"A Loan Estimate tells you important details about a mortgage loan you have requested. Use this tool to make sure it’s accurate—and what you expected."} %}
+{% set active_page = "loan-estimate" %}
+{% set page_js = "loan-estimate" %}
+
+{% block title %}Loan Estimate Explainer{% endblock title %}
+{% block metadescription %}Use the CFPB's tool to review your Loan Estimate to make sure it reflects what you discussed with the lender.{% endblock metadescription %}
+
```

Comment:
Looks like you need to add https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/jinja2/v1/owning-a-home/explore-rates/index.html#L13-L15 to the loan-estimate template (or we could create a `base-oah.html` like you had in the original PR for this).

Reply:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/3824/commits/23dffe797973c631f24e0d999dceba4eadccb850